From e1208b772490bdb7979dca06dcadb17daa6be9a5 Mon Sep 17 00:00:00 2001
From: Andrea Fagiani <andfagiani@gmail.com>
Date: Tue, 18 Oct 2016 09:33:52 +0200
Subject: [PATCH] Remove borders and add horiz/vert padding

---
 sources/PseudoTerminal.m         | 3 +--
 sources/iTermTextDrawingHelper.h | 4 ++--
 2 files changed, 3 insertions(+), 4 deletions(-)

diff --git a/sources/iTermTextDrawingHelper.h b/sources/iTermTextDrawingHelper.h
index 0612aa8..339d7ec 100644
--- a/sources/iTermTextDrawingHelper.h
+++ b/sources/iTermTextDrawingHelper.h
@@ -13,10 +13,10 @@
 #import "VT100GridTypes.h"
 
 // Number of pixels margin on left and right edge.
-#define MARGIN 5
+#define MARGIN HORIZONTAL_VAR
 
 // Number of pixels margin on the top.
-#define VMARGIN 2
+#define VMARGIN VERTICAL_VAR
 
 @class iTermColorMap;
 @class iTermFindOnPageHelper;
-- 
2.8.4 (Apple Git-73)

